recipes/libsrtp/all/conanfile.py

```diff
@@ -0,0 +1,89 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class ConanRecipe(ConanFile):
+    name = "libsrtp"
+    description = (
+        "This package provides an implementation of the Secure Real-time Transport"
+        "Protocol (SRTP), the Universal Security Transform (UST), and a supporting"
+        "cryptographic kernel."
+    )
+    topics = ("conan", "libsrtp", "srtp")
+    homepage = "https://github.com/cisco/libsrtp"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "BSD-3-Clause"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "enable_openssl": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "enable_openssl": False,
+    }
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.cppstd
+        del self.settings.compiler.libcxx
+
+    def requirements(self):
+        if self.options.enable_openssl:
+            self.requires("openssl/1.1.1i")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+        if self.options.shared:
+            tools.replace_in_file(
+                os.path.join(self._source_subfolder, "CMakeLists.txt"),
+                "install(TARGETS srtp2 DESTINATION lib)",
+                (
+                    "install(TARGETS srtp2\n"
+                    "RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}\n"
+                    "LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}\n"
+                    "ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})"
+                ),
+            )
```

Comment:
FYI, you can do this replace unconditional.
The install will place the runtime/library/archive component in the correct folder.
You can even submit this patch upstream (while making sure `GNUInstallDirs` is included).

Reply:
Done, also, `${CMAKE_INSTALL_BINDIR}` gives me `bin` even if there is no `include(GNUInstallDirs)` 🤔 is this a CMake fallback? Anyways, I will add an `include(GNUInstallDirs)` to be on the safe side 👍